components/form/FormItem.tsx

```diff
@@ -26,6 +25,14 @@ export interface FormItemProps {
   colon?: boolean;
 }
 
+function intersperseSpace<T>(list?: Array<T>): Array<T | string> {
```

Comment:
`Array<T | string>` `string` is unnecessary.

Reply:
Since we insert string `' '` in the array. Remove `| string` will get error.